onig_sys/build.rs

```diff
@@ -112,8 +128,10 @@ fn compile(link_type: LinkType) {
 }
 
 pub fn main() {
-    if let Ok(_) = pkg_config::find_library("oniguruma") {
-        return;
+    if env_var_bool("RUSTONIG_SYSTEM_LIBONIG").unwrap_or(true) {
```

Comment:
Not sure about having another flag here. Another option would be to always build Oniguruma ourselves if either `RUSTONIG_DYNAMIC_LIBONIG` or `RUSTONIG_STATIC_LIBONIG` is set?

Or check `CARGO_FEATURE_PRINT_DEBUG` here as well. What do you think?

Reply:
I think it is still useful to have the option to choose how you're linked to the system library. Keep it as two falgs for now.